### PR TITLE
Updated lists 27th of October

### PR DESCRIPTION
**Pull request by RosaWagner**

### CODE REVIEW

**Comment by RosaWagner** [comment]
the lang path are actually valid, I don't know why the CI fails them. cc @m4rc1e

File: to_production.txt [hunk 1]
```diff
@@ -1,5 +1,24 @@
 # New
-ofl/rethinksans # https://github.com/google/fonts/pull/6651 HELD DUE TO SUBSETTER CRASH
```
Comment (chrissimpkins) [inline to_production.txt:2]: Do we know if this subsetter issue has been addressed @rsheeter?

**Comment by chrissimpkins** [comment]
@m4rc1e mind having a look at the CI fail here before we merge?  It's flagging something as broken in the metadata file, but [Rosa thinks that the paths are ok](https://github.com/google/fonts/pull/6921#issuecomment-1786818181).

File: to_production.txt [hunk 2]
```diff
@@ -1,5 +1,24 @@
...
+ofl/notosansarmenian # https://github.com/google/fonts/pull/6759
+ofl/notosanscypriot # https://github.com/google/fonts/pull/5278
+ofl/notosansmeroitic # https://github.com/google/fonts/pull/6775
+ofl/notosanssymbols2 # https://github.com/google/fonts/pull/6861
+ofl/notosansthaana # https://github.com/google/fonts/pull/6792
+ofl/notoserifahom # https://github.com/google/fonts/pull/6756
+ofl/notoserifarmenian # https://github.com/google/fonts/pull/6758
+ofl/notosanshanunoo # https://github.com/google/fonts/pull/6418
+ofl/notoserifdogra # https://github.com/google/fonts/pull/5377
+ofl/robotoflex # https://github.com/google/fonts/pull/6484
```
Comment (chrissimpkins) [inline to_production.txt:16]: What are the changes happening in Roboto Flex here?

**Comment by RosaWagner** [comment]
closing in favour of #6936 
@chrissimpkins I updated the lists with the last addition to dev from this week and the last fixs to the push tool from marc. There is no more CI problem.

**Pull request closed without merging**